VideoLocker/src/main/java/org/edx/mobile/view/dialog/CourseModesMenuPopup.java

```diff
@@ -0,0 +1,92 @@
+package org.edx.mobile.view.dialog;
+
+import android.app.Activity;
+import android.graphics.Color;
+import android.graphics.drawable.ColorDrawable;
+import android.support.annotation.IdRes;
+import android.util.DisplayMetrics;
+import android.util.TypedValue;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.FrameLayout;
+import android.widget.PopupWindow;
+
+import org.edx.mobile.R;
+
+public class CourseModesMenuPopup extends PopupWindow {
+    public enum CourseMode {
+        VIDEO_ONLY(R.id.videos_only),
```

Comment:
Yes, we can create a generic menu if needed, by using `ListPopupWindow` with an adapter.